### PR TITLE
rename daq slider example

### PR DESCRIPTION
**Pull request by cldougl**

### CODE REVIEW

File: tutorial/chapter_index.py [hunk 1]
```diff
@@ -451,7 +451,7 @@
         'description': 'StopButton examples, properties, and reference.'
     },
 
-    'slider-examples': {
+    'daq-slider-examples': {
```
Comment (cldougl) [inline tutorial/chapter_index.py]: this resolves the dcc.Slider doc that isn't rendering at this time 
screen shot 2019-01-23 at 1 41 29 am

File: tests/test_integration.py [hunk 2]
```diff
@@ -115,11 +115,14 @@ def test_docs(self):
         ]
 
         def visit_and_snapshot(href):
-            self.driver.get('http://localhost:8050{}'.format(href))
-            self.wait_for_element_by_id('wait-for-page-{}'.format(href))
-            time.sleep(5)
-            self.snapshot(href)
-            self.driver.back()
+            if href == '/dash-daq':
```
Comment (cldougl) [inline tests/test_integration.py]: suggestion for resolving https://github.com/plotly/dash-docs/issues/359 as this is timing out on the snapshot of https://dash.plot.ly/dash-daq

Increasing the timeout (https://github.com/plotly/dash-docs/blob/master/tests/utils.py#L4) didn't seem to make a difference. 

@shammamah @chriddyp does anyone have any insight into loading issues with the daq page or why `wait_for_element_by_id()` is failing for this page?

**Comment by cldougl** [comment]
leaning towards merging this asap to get tests back into a useful state + dcc slider reference back up and logging an issue for further investigation on the /dash-daq snapshot failure. 
Any objections? cc @Marc-Andre-Rivet

**Comment by chriddyp** [comment]
> does anyone have any insight into loading issues with the daq page or why wait_for_element_by_id() is failing for this page?

To debug, I'd recommend removing `wait_for_`, adding `time.sleep(10)` and then taking a snapshot. Then, manually inspect the snapshot that got generated: will it be blank, or or partially loaded, or will there be an error?

**Comment by chriddyp** [comment]
I'd also recommend running the tests locally: does the page load correctly and does the ID exist? Running the tests locally uses a different version of the bundles I think (locally vs on cdn), so the page will appear different than if you were to just run the app locally.

**Comment by chriddyp** [comment]
> Any objections?

No objections from me.

**Comment by shammamah** [comment]
💃 Good to have the tests back!

**Comment by cldougl** [comment]
follow up issue https://github.com/plotly/dash-docs/issues/377

**Pull request merged**